internal/provider/resource_automation_rule_jira_add_comment.go

```diff
@@ -0,0 +1,385 @@
+
+	"wiz.io/hashicorp/terraform-provider-wiz/internal"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/client"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/utils"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/wiz"
+)
+
+func resourceWizAutomationRuleJiraAddComment() *schema.Resource {
+	return &schema.Resource{
+		Description: "Automation Rules define associations between actions and findings.",
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Description: "Wiz internal identifier.",
+				Computed:    true,
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The date/time at which the automation rule was created.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the automation rule",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Description of the automation rule",
+			},
+			"trigger_source": {
+				Type:     schema.TypeString,
+				Required: true,
+				Description: fmt.Sprintf(
+					"Trigger source.\n    - Allowed values: %s",
+					utils.SliceOfStringToMDUList(
+						wiz.AutomationRuleTriggerSource,
+					),
+				),
+				ValidateDiagFunc: validation.ToDiagFunc(
+					validation.StringInSlice(
+						wiz.AutomationRuleTriggerSource,
+						false,
+					),
+				),
+			},
+			"trigger_type": {
+				Type:     schema.TypeList,
+				Required: true,
+				Description: fmt.Sprintf(
+					"Trigger type. Must be set to `CREATED` for wiz_automation_rule_jira_add_comment.\n    - Allowed values: %s",
+					utils.SliceOfStringToMDUList(
+						wiz.AutomationRuleTriggerType,
+					),
+				),
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					ValidateDiagFunc: validation.ToDiagFunc(
+						validation.StringInSlice(
+							wiz.AutomationRuleTriggerType,
+							false,
+						),
+					),
+				},
+			},
+			"filters": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateDiagFunc: validation.ToDiagFunc(
+					validation.StringIsJSON,
+				),
+				Description: "Value should be wrapped in jsonencode() to avoid diff detection. This is required even though the API states it is not required.  Validate is performed by the UI.",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Enabled?",
+				Default:     true,
+			},
+			"project_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Wiz internal ID for a project.",
+			},
+			"action_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Wiz internal ID for the action.",
+			},
+			"integration_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Wiz identifier for the Integration to leverage for this action. Must be resource type integration_aws_sns.",
```

Comment:
must be jira integration type?

Reply:
fixing this in all 3, foiled by pasting, ahhhhh 😱